manifests/profile.pp

```diff
@@ -148,6 +157,10 @@
     fail("Duplicity::Profile[${title}]: max_full_backups must be an integer, got '${max_full_backups}'")
   }
 
+  if "str${max_fulls_with_incrs}" !~ /^str[0-9]*$/ {
```

Comment:
re: `"str${..}"`

I'm not familiar with this syntax; can you please link me some documentation that explains this?